bins/ggipcd/src/services/config/subscribe_to_configuration_update.c

```diff
@@ -109,6 +118,10 @@ GglError ggl_handle_subscribe_to_configuration_update(
     } else {
         ret = ggl_ipc_get_component_name(handle, &component_name);
         if (ret != GGL_ERR_OK) {
+            *ipc_error = (GglIpcError) {
+                .error_code = GGL_IPC_ERR_SERVICE_ERROR,
+                .message = GGL_STR("Failed to get component name from handle.")
```

Comment:
Why not log this error like the others?

Reply:
This isn't actually useful here, since if we can't get component name, we wont return a response. 
Only reason this happens is if connection is lost. Not really an error since client can close any time. We should just exit.

Reply:
This should also just return FATAL probably (If we do log, should be DEBUG or TRACE).

Reply:
I updated the code to not use ggl_ipc_get_component_name